packaging/standalone/src/main/distribution/shell-scripts/bin/Neo4j-Management/Get-Java.ps1

```diff
@@ -146,47 +151,28 @@ Function Get-Java
     if ($javaPath -eq '') { Write-Error "Unable to determine the path to java.exe"; return $null }
     if ($javaCMD -eq '') { $javaCMD = "$javaPath\bin\java.exe" }
     if (-not (Test-Path -Path $javaCMD)) { Write-Error "Could not find java at $javaCMD"; return $null }
-  
+ 
     $ShellArgs = @()
 
+    Write-Verbose "Java detected at '$javaCMD'"
+    Write-Verbose "Java version detected as '$javaVersion'"
```

Comment:
Is the setting of `$javaVersion` an implementation detail side effect of getting Java from the registry. It's not set at all if we just get passed `JAVA_HOME`, which makes this look odd (and even more the warning that we're unable to determine it.